shimmingtoolbox/shim/sequencer.py

```diff
@@ -937,13 +1001,13 @@ def select_optimizer(method, unshimmed, affine, coils: ListCoil, opt_criteria, p
 
 
 def _optimize(optimizer: Optimizer, nii_mask_anat, slices_anat, shimwise_bounds=None,
-              dilation_kernel='sphere', dilation_size=3, path_output=None):
+              dilation_kernel='sphere', dilation_size=3, w_signal_loss=0, epi_te=0, path_output=None):
```

Comment:
I don't think you need to add the options when calling `_optimize`. The `optimizer` already contains the appropriate variables since it was initialized in the `select_optimizer` function.

Reply:
Keep it as it is since removing these two variables seems to give me errors